lib/plottr_components/src/connections/pltr.js

```diff
@@ -129,6 +129,7 @@ const pltrTypeSpecs = {
     readFileSync: PropTypes.func.isRequired,
     moveItemToTrash: PropTypes.func.isRequired,
     createFromSnowflake: PropTypes.func.isRequired,
+    // createFromScivener: PropTypes.func.isRequired,
```

Comment:
Please uncomment this line.
It's an important check to avoid run time errors in prod.